data/strings/en.ini

```diff
@@ -1900,3 +1906,9 @@ toggle_left_diagonal = Toggle -45° Symmetry
 show_options = Symmetry Options
 reset_position = Reset Symmetry to Center
 reset_position_to_view_center = Reset Symmetry to View Center
+
+[textedit]
+copy = &Copy
+cut = Cu&t
+paste = &Paste
+select_all = Select &All
```

Comment:
This is a good idea, I'll try to add these strings to `en.ini` and start using them in `ui::Entry` in some way (so then you can move `TextEdit` to the `ui-lib`).